src/scripts/components/lang-redirects.js

```diff
@@ -0,0 +1,100 @@
+import Cookies from 'js-cookie';
+
+const allowedLanguages = ['en', 'ja', 'fr'];
+const enabledSubdomains = [
+	'docs',
+	'docs-staging',
+	'corpsite-preview',
+	'corpsite-staging',
+	'localhost'
+];
+const cookiePath = '/';
+
+const getUrlVars = () => {
+	const vars = {};
+	const href = window.location.href.replace(window.location.hash, '');
+    href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+        vars[key] = value;
+    });
+    return vars;
+}
+
+const getDomain = (data) => {
+    const a = document.createElement('a');
+    a.href = data;
+    return a.hostname;
+}
+
+const getPath = (data) => {
+    const a = document.createElement('a');
+    a.href = data;
+    return a.pathname;
+}
```

Comment:
I was thinking you might want to condense these 2 into one function that returns both? I guess its a preference kind of thing if you think its easier to read / maintain together or seperate

```suggestion:-11
const getUrlParts = (data) => {
    const a = document.createElement('a');
    a.href = data;
    return {hostname: a.hostname, pathname: a.pathname};
}
```

then later you could just do

`const {hostname, pathname} = getUrlParts(window.location.href);`

Reply:
+1 good suggestion, I've updated the function